.travis.yml

```diff
@@ -19,6 +21,13 @@ env:
   - GCLOUD=$HOME/gcloud/google-cloud-sdk/bin 
   - CLOUDSDK_PYTHON_SITEPACKAGES=1 
   - GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/servicekey.json
+  #GenomicsDB Stuff
+  - GATK_GENOMICSDB_BIN=src/test/resources/large/tiledb_bin
```

Comment:
@droazen : As @lbergelson mentioned, there are two types of binaries being used here:

1. Shared library packaged inside the GenomicsDB jar - this binary is portable (dependencies statically linked in as much as possible) across different versions of Linux and Macs and is distributed on Maven central. Users who build GATK will pull this jar file as part of the build process.
1. Executables packaged inside the GATK repo - these executables are for GATK-GenomicsDB CI testing in Travis only. The GATK repo could pack pre-loaded TileDB test arrays; however, TileDB requires that the full absolute path of arrays be the same on the loader system and the query system. Hence, Louis had to distribute the GenomicsDB binaries and load TileDB test arrays from the test gVCFs in Travis.
The second set of binaries are not portable (since they depend on MPI, the specific gcc compiler used etc) - we have no means to distribute portable versions of these executables. However, most users will/should *NOT* be using these binaries directly from the GATK repo - they should follow instructions from the GenomicsDB wiki (in the future, we might distribute platform specific rpms/dpkgs with the dependencies enumerated).